app/HMS/Helpers/HmsHelper.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace HMS\Helpers;
+
+use Symfony\Component\HttpFoundation\IpUtils;
+
+/**
+ * Place to keep general helper that will be available via the HmsHelper:: facade.
+ */
+class HmsHelper
```

Comment:
In this case we could keep the helpers to a single piece of functionality each (SOLID in action right there) - call it HMS\Helpers\InTheSpace

Having this in place means that people have to think really hard about where stuff should go rather then dumping a new function in HMSHelper.